adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -282,11 +318,15 @@ public String getErrorLogInfo() {
      * @return true if expired
      */
     public boolean isExpired() {
```

Comment:
Sorry I don't quite understand the check. How do you know if the outgage mode is turned on in the AuthenticationResult class iteself? If it's not turned on, why we're checking extended expires on?